KB and images to Verify IPS from the command line

mclose opened the pull request:
KB covers how an user of CLC IPS can verify that the IPS is functioning and they have properly configured notifications.

chrislittle commented on Security/verify_IPS_from_the_command_line.md:
```diff
@@ -0,0 +1,158 @@
+
+**Note:** It is possible to complete this test without installing ruby, sinatra, and ngrok by using other HTTP request inspectors. For example, you could use [requestb.in](http://requestb.in). If you choose to use another inspector, you can skip to the second section. However, please be sure to use the URL provided by [requestb.in](http://requestb.in) instead of the ngrok URL.
+
+#### Instructions for setting up your webhook listener
```
can we simplify this at all?  there are ample free services so customers don't have to setup a whole webhook listener just to verify the service is functional.  zapier.com for example could be had in 2 minutes and a url to listen on.  just seems like quite a bit of technical work for a test event.

mclose replied:
Okay, I'll reduce it down with either requestb.in or zapier. Part of the intention was to be able to have events processed in house, but I do understand your point. This KB is probably too long for most folks.

chrislittle replied:
i'm not married to it.  just thinking from an ease of use view.  if you want to leave it in for advanced users perhaps just have the 2 options, put in a little table of contents to the kb and people can flip to which makes sense.  on some of my longer in depth kbs there isn't always 1 approach.

nathanyoung commented on Security/verify_IPS_from_the_command_line.md:
```diff
@@ -0,0 +1,158 @@
+  "sticky": false
+}}}
+
+### Verify IPS Install from the Command Line
```
remove this line. The title is generated from the front-matter.

mclose replied:
Will do.

mclose commented:
Okay, I've simplified the KB article and made the necessary edits to format. Switched from full ruby/gem route to just using requestb.in. (But if you haven't used ngrok before, it's still a great tool) I looked into using Zapier for this KB, but Zaps with webhooks require the format of the POST to be set before you can use them. We intend to change the notification format fairly soon so I didn't want to have to alter this KB as a result. 

When y'all get a chance can you review again?  Thank you!

chrislittle commented:
@nathanyoung i'm ok with the technical content updates I noted previously.
